app/src/pages/Play/operations.graphql

```diff
@@ -9,10 +9,10 @@ mutation CreateTurn($gameId: uuid!, $playerId: uuid!, $roundId: uuid!) {
   }
 }
 
-mutation StartTurn($currentTurnId: uuid!, $startedAt: timestamptz!) {
+mutation StartTurn($currentTurnId: uuid!) {
   update_turns_by_pk(
     pk_columns: { id: $currentTurnId }
-    _set: { started_at: $startedAt }
+    _set: { started_at: "now()" }
```

Comment:
Most of these game state mutations now leverage this approach which will use the server's `now()` in favor of the client's to mitigate clock drift.

This approach also seems to add more precision, i.e. a timestamp before:

```
2020-12-23T21:36:12.328+00:00
```

and after:

```
2020-12-23T23:15:53.779141+00:00
```

I moved all client-side `Date` object construction to the [core constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) (which was looser than the [`date-fns` stuff](https://github.com/avimoondra/fishbowl/pull/124/files#diff-335b37efa55c8a5da19033a42fc03cde988a2d337a073cb70a38db7c4add805dL12) we had going) to handle this change. This seems fine, but let me know if there are concerns.